_Streams/121-map-reduce-in-javascript.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "map() and reduce() in JavaScript"
+video_number: 121
+date: 2018-02-16
+video_id: kWoZ9_ECZxQ
+
+topics:
+ - title: "Introduction to higher order functions"
+   time: "19:04"
+ - title: "map()"
+   time: "44:00"
+ - title: "reduce()"
+   time: "1:09:53"
+   
+links:
+ - title: "Arrays on MDN"
+   url: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array"
+ - title: "Matter.js"
+   url: "http://brm.io/matter-js/"
+
+videos:
+  - title: "Foundations of Programming in JavaScript"
+    url: "https://www.youtube.com/playlist?list=PLRqwX-V7Uu6Zy51Q-x9tMWIv9cueOFTFA"
+  - title: "CircleCI: Unit Testing and CI"
+    url: "https://www.youtube.com/watch?v=CB7vnoXI0pE"
+  - title: "ES6 Arrow Function"
+    url: "https://www.youtube.com/watch?v=mrYMzpbFz18"
+  - title: "Closure in JavaScript"
+    url: "https://www.youtube.com/watch?v=-jysK0nlz7A"
+  - title: "Prototypes in JavaScript"
+    url: "https://www.youtube.com/watch?v=hS_WqkyUah8"
+---
+
+In this live stream, I cover higher order functions in JavaScript and discuss them in the context of arrays: map() and reduce()
```

Comment:
You may want to use proper markdown formatting for the function names (like it was in `_Streams/121-upcoming.md`)